### ISSUE TITLE
Base64 codec compilation problem

### ISSUE THREAD

**Issue by l15k4** [lihaoyi/roll#3]
I created a pull request, but I forgot to add `closes #3` so it's in a #4 issue

```
[error] ~/src/scalajs/roll/project/Build.scala:6: object commons is not a member of package org.apache
[error] import org.apache.commons.codec.binary.Base64
[error]                   ^
[error] ~/src/scalajs/roll/project/Build.scala:33: not found: value Base64
[error]             val data = Base64.encodeBase64String(IO.readBytes(inFiles.head)).replaceAll("\\s", "")
[error]                        ^
[error] two errors found
```

**Comment by Tasselmi** [comment]
I have the same problem， did you solved it ？

**Comment by l15k4** [comment]
IDK, I thought it was fixed by that #4
